### DISCUSSION TITLE
Python README requires minor update for AzureChatCompletion

### DISCUSSION THREAD

**Discussion by LandMagic** [microsoft/semantic-kernel#4435] [General]
The current README.md at:
https://github.com/microsoft/semantic-kernel/blob/main/python/README.md

has this:
```
# Alternative using Azure:
# deployment, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
# kernel.add_chat_service("dv", AzureChatCompletion(deployment, endpoint, api_key))
```

However, the only way I could make this work was by doing:
```
kernel.add_chat_service("dv", AzureChatCompletion(deployment_name=deployment, endpoint=endpoint, api_key=api_key))
```

a small, but essential difference that I found in the semantic tutorials. Maybe there was another intention at the time of creating the readme? If so, some clarification might be helpful. Thanks!

**Comment by evchaki** [comment]
@moonbox3  can you take a look at this?
